setup.py

```diff
@@ -43,4 +43,7 @@ def read(fname):
     # A list of all available classifiers can be found at 
     # https://pypi.python.org/pypi?%3Aaction=list_classifiers
     install_requires=INSTALL_REQUIRES,
+    extras_require={    
+        "torch": ["torch"],     # optional for torch cnn wd, install with pycomlink[torch]
```

Comment:
did you test this? it is pytorch on conda forge and torch using pip

CC @cchwala

Reply:
I have no experience with using configurations like that. IMO the benefit is small since the only change for the user is that pytorch is installed automatically, but you have to remember to do `pip install pycomlink[torch]` instead of just doing `pip install pycomlink torch`. Hence, I would tend to leave this out. 

We have to switch to pyproject.toml anyway. Maybe we can split things up into different options of pycomlink, but, again, in general, I do not see a large benefit, but added complexity.

Reply:
The problem is, we will only be able to find out when doign a releaes on pypi, which I have to do manually because there is no autmatic action for that (like modern github python projects should have...).

Hence, yes, please remove it.